src/box/lua/execute.c

```diff
@@ -346,7 +346,8 @@ lua_sql_bind_decode(struct lua_State *L, struct sql_bind *bind, int idx, int i)
 		 * will be poped from Lua stack.
 		 */
```

Comment:
Please expand in NO_TEST the description of the test that tests this patch. I, for example, am not sure what this test is.

Reply:
I tried to describe it in a little more detail.